api/src/main/java/com/exelaration/abstractmemery/services/FileStorageService.java

```diff
@@ -0,0 +1,13 @@
+package com.exelaration.abstractmemery.services;
+
+import com.exelaration.abstractmemery.domains.Image;
+import org.springframework.web.multipart.MultipartFile;
+
+public interface FileStorageService {
+
+  public static final String uploadingDir = "/app/src/main/resources/images/";
```

Comment:
I don't think this belongs on the interface.  This is a bit nonsensical if we're using a storage service that isn't our local file system.